Source/DafnyCore/Compilers/Compiler-java.cs

```diff
@@ -2361,41 +2361,50 @@ private class GenericArrayElementLvalue : ILvalue {
         }
       }
 
-      // Find all .class files
-      var classfiles = new List<String>();
-      foreach (string file in Directory.EnumerateFiles(targetDirectory, "*.class", SearchOption.AllDirectories)) {
-        classfiles.Add(Path.GetRelativePath(targetDirectory, file));
-      }
+      var classFiles = Directory.EnumerateFiles(targetDirectory, "*.class", SearchOption.AllDirectories)
```

Comment:
Why do we want the paths in `classFiles` to be relative?

Reply:
I think the jar file internal structure is not correct otherwise. At leat that is how I've always created jar files.